js/src/personal/maliha/ModalMenu.tsx

```diff
@@ -33,24 +38,20 @@ export function Menu() {
         opened={opened}
         onClose={handleToggleMenu}
         title="Menu"
-        size={400}
+        size={450}
         zIndex={2}
+        pl={20}
         overlayProps={{
           backgroundOpacity: 0.1,
           blur: 0,
         }}
       >
-        <Stack
-          w={100}
-          miw={100}
-        >
-          {items}
-        </Stack>
+        <Stack>{items}</Stack>
```

Comment:
nit: "items" pretty vague if there are more descriptive variable names like "menuButtons", but since this is purely local it's less big of deal